docs/release-notes/5.7.41-44.md

```diff
@@ -0,0 +1,68 @@
+# Percona Server for MySQL 5.7.41-44 (2023-02-28)
+
+<style>1
+    table {
+        width=50%';
+        font-family: Poppins;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+    tr:hover {
+        background-color: #dddd;
+    }
+</style>
+
+| Release date | February 28, 2023 |
+|---|---|
+| Install instructions | [Install Percona Server for MySQL](https://www.percona.com/doc/percona-server/5.7/installation.html) |
+| Download this version | [Percona Server for MySQL](https://www.percona.com/downloads/Percona-Server-5.7/LATEST/)
+
+[Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) 5.7.41-44
+includes all the features and bug fixes available in [MySQL 5.7.41 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-41.html) in addition to enterprise-grade features developed by Percona.
+
+Percona Server for MySQL is a free, fully compatible, enhanced, and open
+source drop-in replacement for any MySQL database. It provides superior
+performance, scalability, and instrumentation.
+
+Percona Server for MySQL is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads. It
+delivers more value to MySQL server users with optimized performance,
+greater performance scalability and availability, enhanced backups, and
+increased visibility.
+
+For paid [support](https://www.percona.com/services/support), [managed services](https://www.percona.com/services/managed-services) or [consulting services](https://www.percona.com/services/consulting), contact [Percona Sales](https://www.percona.com/about-percona/contact)
+
+For [training](https://www.percona.com/training), contact [Percona Training - Start learning now](https://learn.percona.com/contact-me)
+
+## Release highlights
+
+Improvements and bug fixes provided by Oracle for MySQL 5.7.41 and included in Percona Server for MySQL are the following:
+
+* Updated the linked OpenSSL library for MySQL Server to 1.1.s
+  
+* Updated the bundled zlib library to zlib 1.2.13. This zlib library version is the minimum supported.
+
+* While the SQL thread handled a transaction, issuing `STOP_SLAVE_SQL_THREAD` caused replication to stop immediately instead of waiting for the event group to complete before the shutdown.
+  
+* When the `--single-transaction` and `--set-gtid-purged=ON` options were both used, the mysqldump backup of the data and GTIDs were inconsistent. Now at the beginning of the dump, a `FLUSH_TABLES_WITH_READ_LOCK` and `GTID_EXECUTED` are performed. These operations ensure that the GTID value is consistent with the mysqldump snapshot. 
```

Comment:
@Partick We deliberately removed this Oracle's patch from the 5.7.41 as it requires additional (not really needed) privileges for users who do not use GTID-based replication. Instead we left our (Percona-specific) fix for the original problem that does not have this regression.
We basically reverted
https://github.com/mysql/mysql-server/commit/72f28e5ba7838beee4e5eabcf1118024b7b38b2a
and
https://github.com/mysql/mysql-server/commit/dd7bfa81ce663e784db15c4eea05681552eab47b
and left our solution based on `START TRANSACTION WITH CONSISTENT SNAPSHOT`.
You should also refer the following two bugs submitted to Oracle
https://bugs.mysql.com/bug.php?id=109701
and
https://bugs.mysql.com/bug.php?id=109685

Reply:
@oleksandr-kachan Please, confirm that this is what has been done for 5.7 as well.

Reply:
It looks like Oracle admitted the problem and fixed this in 5.7.42 / 8.0.33
```
5.7.42:
With mysqldump, using --single-transaction required either the RELOAD or
FLUSH_TABLES privilege with mysqldump v5.7.41. This requirement now only
applies when gtid_mode=ON (defaults to OFF) and with --set-gtid-purged =
ON|AUTO (defaults to AUTO).

8.0.33:
With mysqldump, using --single-transaction required either the RELOAD or
FLUSH_TABLES privilege with mysqldump v8.0.32. This requirement now only
applies when gtid_mode=ON (defaults to OFF) and with --set-gtid-purged =
ON|AUTO (defaults to AUTO).
```

Reply:
Added